demos/python_demos/llm_text_generation/README.md

```diff
@@ -56,9 +69,12 @@ Completion:
  Many jobs in the health care industry are experiencing long-term shortages due to a lack of workers, while other areas face overwhelming stress and strain.  Due to COVID-19 many more people look for quality medical services closer to home so hospitals have seen record levels of admissions over the last year.
 ```
 
-## Deploying the streaming version
+## Requesting the LLM with gRPC streaming
+
 
-Restart the Model Server with different directory mounted (`./servable_stream_version`):
+Start the Model Server with different directory mounted (`./servable_stream_version`).
+It contains modified `model.py` script which yields the intermediate results instead of returning it at the end of `def execute()` method.
```

Comment:
def execute() -> execute

no need for this def detail + the signature is not valid, let's stick to just method name

Reply:
done